src/st/no2/33D8C.c

```diff
@@ -49,8 +49,8 @@ void func_us_801B3D8C(Entity* self) {
 }
 
 /* st/no2/data/B7C.data.s */
-u16 D_us_80180B7C[] = {0x8044, 0x8048, 0x8049, 0x804A, 0x804B, 0x804C, 0x804D, 0x0000};
-u32 D_us_80180B8C[] = {0x4, 0x8, 0x9, 0xA, 0xB, 0xC, 0xD};
```

Comment:
We can go through it move in Discord if you like, but if you want to import data, you need to exclude the segment from splat, otherwise the data effectively gets included in the bin twice which won't match.

You would do something to the effect of
```
      - [0xB7C, .data, 33D8C]
      - [0xBA8, data]
```

<img width="466" height="571" alt="image" src="https://github.com/user-attachments/assets/84a62f7d-4ec2-41ae-974f-70b9f0cf1265" />

But we don't require importing data right away and this can be done later on